WordPress/Sniffs/CSRF/NonceVerificationSniff.php

```diff
@@ -22,21 +22,23 @@ class WordPress_Sniffs_CSRF_NonceVerificationSniff extends WordPress_Sniff {
 	 * Superglobals to give an error for when not accompanied by an nonce check.
 	 *
 	 * @since 0.5.0
+	 * @since 0.11.0 Changed visibility from public to protected.
 	 *
 	 * @var array
 	 */
-	public $errorForSuperGlobals = array( '$_POST', '$_FILE' );
+	protected $errorForSuperGlobals = array( '$_POST', '$_FILE' );
```

Comment:
Got it. Thanks for explaining the history. I'm open to leaving these `public` and adding them to the upcoming wiki page if you prefer.